dev/tasks/verify-rc-alma/action.yml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We don't need to create a new directory for this. We can reuse `/dev/tasks/verify-rc/` directory.

Reply:
Ok